pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -10,20 +10,14 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "meson";
-  version = "0.61.2";
+  version = "0.62.1";
 
   src = python3.pkgs.fetchPypi {
     inherit pname version;
-    hash = "sha256-AjOn+NlZB5MY9gUrCTnCf2il3oa6YB8lye5oaftfWIk=";
+    sha256 = "sha256-oPXKoecNoS1eY6pqlQQnN1m4ka82yNh944Gk7ROA6EU=";
   };
 
   patches = [
-    # Upstream insists on not allowing bindir and other dir options
-    # outside of prefix for some reason:
-    # https://github.com/mesonbuild/meson/issues/2561
-    # We remove the check so multiple outputs can work sanely.
-    ./allow-dirs-outside-of-prefix.patch
```

Comment:
The file should be removed too.

Reply:
Ah yep thanks, removed

Reply:
Looks like it is back?

Reply:
Thanks, I must have botched a rebase there. Removed again